section2/exercises/if_statements.rb

```diff
@@ -61,5 +68,17 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
-has_sauce = true
+cups_of_flour = 2
+has_sauce = false
+
+puts "How much flour do you have?"
+  if cups_of_flour < 2
```

Comment:
Is there a way you could write this logic so that lines 76 and 80 (which are the exact same) only need to be typed one time?